src/index.js

```diff
@@ -11,6 +11,8 @@ import rootSaga from './Root/rootSaga';
 import configureStore from './store/configureStore';
 import Root from './Root/Root';
 
+import '../node_modules/grommet/grommet.min.css';
```

Comment:
Talking about native environment support, in that case jsx will still require webpack support. While js don't. If jsx files use js extension, by that time it's gonna be an effort to rename those so webpack loader could correctly find and transpile jsx files.